app/src/main/java/com/framgia/forder/screen/shopinfo/ShopinfoViewModel.java

```diff
@@ -2,6 +2,7 @@
 
 import android.databinding.BaseObservable;
 import com.framgia.forder.R;
+import com.framgia.forder.data.model.Domain;
```

Comment:
ERROR: Unused import: com.framgia.forder.data.model.Domain. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports